examples/psp22-extension/README.md

```diff
@@ -0,0 +1,39 @@
+# PSP22 Chain Extension Example
+
+## What is this example about?
+
+It is an example implementation of the [PSP22 Fungible Token Standard](https://github.com/w3f/PSPs/blob/master/PSPs/psp-22.md) as a chain extension, supporting a multi-token system provided by the [FRAME assets pallet](https://docs.substrate.io/rustdocs/latest/pallet_assets/index.html). It effectively allows ink! contracts (L2) to interact with native assets (L1) from the chain runtime in a standardized way.
+
+See [this chapter](https://paritytech.github.io/ink-docs/macros-attributes/chain-extension)
+in our ink! documentation for more details about chain extensions.
+
+There are two parts to this example:
+
+* Defining and calling the extension in ink!.
+* Defining the extension in Substrate.
+
+## Chain-side Integration
+
+To integrate this example into Substrate you need to do two things:
+
+* Use the code in [`psp22-extension-example.rs`](runtime/psp22-extension-example.rs)
+  as an implementation for the trait `ChainExtension` in Substrate.
+  You can just copy/paste the content of that file into e.g. your `runtime/src/lib.rs`.
```

Comment:
@HCastano So the text here should advise to create a chain extensions module and copy the code there instead of `lib.rs`?

BTW, I don't know much about chain extensions, but is there a good reason for this code to be copied rather than imported?

Reply:
I think the original intent with the copy/paste bit was that since this was an example, you could copy it and use that as a base to work off of